### PR TITLE
fix: typos in zh translations

### PR DESCRIPTION
**Pull request by ChungZH**

### CODE REVIEW

**Comment by ChungZH** [comment]
wait a minute

File: content/docs/preferences/general/_index.zh.md [hunk 1]
```diff
@@ -15,6 +15,6 @@ weight: 1000
 
 [正则表达式](https://baike.baidu.com/item/正则表达式) 用于匹配字符串中的模式以及查找和替换子字符串。
 
-你可以找到许多在线正则表达式课程，例如：[正则表达式30分钟入门教程](https://deerchao.cn/tutorials/regex/regex.htm)。.
+你可以找到许多在线正则表达式课程，例如：[正则表达式 30 分钟入门教程](https://deerchao.cn/tutorials/regex/regex.htm)。.
```
Comment (ouuan) [inline content/docs/preferences/general/_index.zh.md:18]:
```suggestion
你可以找到许多在线正则表达式课程，例如：[正则表达式 30 分钟入门教程](https://deerchao.cn/tutorials/regex/regex.htm)。
```

**Comment by ChungZH** [comment]
@ouuan 😃

File: content/docs/tips/_index.zh.md [hunk 2]
```diff
@@ -6,23 +6,23 @@ description: 一些可能会有用的提示
...
+除了单步调试，最常用的调试方法就是输出一些变量。你还记得你试图分辨调试输出和正常输出的那些日子吗？使用 CP Editor，你就不需要这样做了。
 
 关键在于：使用 stderr 来输出调试信息。在 C++ 中，这意味着在输出时使用 `cerr` 而不是 `cout`。CP Editor 会将你的程序的 stderr 展示在“消息”区域。
 
 ## 只在选中的测试点上运行
 
 在 CP Editor 中，你不需要总是在所有测试点上运行。
 
-为了告诉 CP Editor 你不想在某个测试点上运行，你只需要取消选中这个测试点。你还可以使用“更多”菜单来管理勾选情况：你可以全选，取消全选，取消选择已通过的（当你通过了大多数测试点而只想在少数未通过的测试点上调试时非常有用）以及反选。
+为了告诉 CP Editor 你不想在某个测试点上运行，你只需要取消选中这个测试点。你还可以使用“更多”菜单来管理勾选情况：全选，取消全选，取消选择已通过的（当你通过了大多数测试点，而只想在少数未通过的测试点上调试时，非常有用）以及反选。
```
Comment (ouuan) [inline content/docs/tips/_index.zh.md:19]:
```suggestion
为了告诉 CP Editor 你不想在某个测试点上运行，你只需要取消选中这个测试点。你还可以使用“更多”菜单来管理勾选情况：全选，取消全选，取消选择已通过的（当你通过了大多数测试点，而只想在少数未通过的测试点上调试时，这非常有用）以及反选。
```

File: content/docs/tips/_index.zh.md [hunk 3]
```diff
@@ -6,23 +6,23 @@ description: 一些可能会有用的提示
...
 
 如果你想在单个测试点上运行，你还可以点击位于测试点的“运行”按钮。
 
 ## 解决有多解的题目时，将输出复制到答案
 
-有时你会解决一些有多个正确答案的题目。OJ 有 Special Judge，但你没有，你也没有时间自己写一个。让这些测试点保持 WA 状态可能让人看着不爽，甚至带来不便，因为你在修改代码后需要再次检查它们是否正确。
+有时你会解决一些有多个正确答案的题目。OJ 有 Special Judge，但你没有，也没有时间自己写一个。让这些测试点保持 WA 状态可能让人看着不爽，甚至带来不便，因为你在修改代码后需要再次检查它们是否正确。
```
Comment (ouuan) [inline content/docs/tips/_index.zh.md:25]:
```suggestion
有时你会解决一些有多个正确答案的题目。OJ 有 Special Judge，但你没有，你也没有时间自己写一个。让这些测试点保持 WA 状态可能让人看着不爽，甚至带来不便，因为你在修改代码后需要再次检查它们是否正确。
```

Comment (ouuan) [reply content/docs/tips/_index.zh.md:25]: 或者
```suggestion
有时你会解决一些有多个正确答案的题目。OJ 有 Special Judge，但你没有 Special Judge，也没有时间自己写一个。让这些测试点保持 WA 状态可能让人看着不爽，甚至带来不便，因为你在修改代码后需要再次检查它们是否正确。
```

**Review by ouuan — APPROVED** [review]

**Comment by ouuan** [comment]
> ** [ouuan](/ouuan) ** approved these changes on behalf of [cpeditor/zh-cn-translators](/orgs/cpeditor/teams/zh-cn-translators) [ 44秒钟前](#pullrequestreview-693294390)

:thinking:

**Review by swift-zym — APPROVED** [review]

File: content/docs/tips/_index.zh.md [hunk 4]
```diff
@@ -6,23 +6,23 @@ description: 一些可能会有用的提示
...
+除了单步调试，最常用的调试方法就是输出一些变量。你还记得你试图分辨调试输出和正常输出的那些日子吗？使用 CP Editor，你就不需要这样做了。
 
 关键在于：使用 stderr 来输出调试信息。在 C++ 中，这意味着在输出时使用 `cerr` 而不是 `cout`。CP Editor 会将你的程序的 stderr 展示在“消息”区域。
 
 ## 只在选中的测试点上运行
 
 在 CP Editor 中，你不需要总是在所有测试点上运行。
 
-为了告诉 CP Editor 你不想在某个测试点上运行，你只需要取消选中这个测试点。你还可以使用“更多”菜单来管理勾选情况：你可以全选，取消全选，取消选择已通过的（当你通过了大多数测试点而只想在少数未通过的测试点上调试时非常有用）以及反选。
+为了告诉 CP Editor 你不想在某个测试点上运行，你只需要取消选中这个测试点。你还可以使用“更多”菜单来管理勾选情况：全选，取消全选，取消选择已通过的（当你通过了大多数测试点，而只想在少数未通过的测试点上调试时，这非常有用）以及反选。
```
Comment (neko-para) [inline content/docs/tips/_index.zh.md:19]: > 这在你通过了大多数测试点，而只想在少数未通过的测试点上调试时非常有用

**Comment by neko-para** [comment]
🤔

**Pull request merged**